helix-view/src/base64/mod.rs

```diff
@@ -0,0 +1,135 @@
+use std::ops::{BitAnd, BitOr, Shl, Shr};
+
+const PAD_BYTE: u8 = b'=';
```

Comment:
It looks like there are a few compliation warnings for Windows since base64 isn't called when compiled on Windows. Making this module `pub` is ok I think